app/views/admin/world_location_news/_world_location_table.html.erb

```diff
@@ -1,47 +1,52 @@
-<h2><%= title %></h1>
+<%= render "govuk_publishing_components/components/heading", {
+  text: title
+} %>
 
-<table id="world-locations" class="table table-striped table-bordered">
-  <thead>
-    <tr class="table-header">
-      <th width="10%">Type</th>
-      <th width="60%">Location</th>
-      <th width="10%">Status</th>
-      <th width="30%">Translations</th>
-    </tr>
-  </thead>
+<%= render "govuk_publishing_components/components/table", {
+  head: [
+    {
+      text: "Type"
+    },
+    {
+      text: "Location"
+    },
+    {
+      text: "Status"
+    },
+    {
+      text: "Translations"
+    }
+  ],
+  rows:
+    world_locations.map do |world_location|
+      [
+        {
+          text: world_location.display_type
+        },
+        {
+          text:
+            link_to(world_location.name, [:admin, world_location.world_location_news], class: "govuk-link") << "<br><br>".html_safe <<
```

Comment:
This definitely works, but maybe something like this is a little easier to parse 

```
tag.p(link_to(world_location.name, [:admin, world_location.world_location_news], class: "govuk-link"), class: "govuk-body govuk-!-margin-bottom-4") +
              if world_location.world_location_news.mission_statement.present?
                format_with_html_line_breaks(world_location.world_location_news.mission_statement)
              else
                sanitize("<span><em>No mission statement yet, </em>#{link_to("create one", edit_admin_world_location_news_path(world_location.world_location_news), class: "govuk-link")}.")
              end
```

Neither of the cases you called html_safe rang any alarm bells in terms of introducing xss vulnerabilities though. Generally i use `sanitize` where possible instead though https://apidock.com/rails/ActionView/Helpers/SanitizeHelper/sanitize

Reply:
Nice, thanks David. 👍 Much better code. I've implemented as suggested.